### PR TITLE
Collection identifiers fixes

### PR DESCRIPTION
**Pull request by mikeger**
- Fixed localytics events identifiers
- Added action buttons identifiers for image view

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift [hunk 1]
```diff
@@ -91,18 +91,22 @@ internal final class ConversationImagesViewController: UIViewController {
     private func createControlsBar() {
         let copyButton = IconButton.iconButtonDefault()
         copyButton.setIcon(.copy, with: .tiny, for: .normal)
+        copyButton.accessibilityLabel = "copy"
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift]: Should not be `copyButton.accessibilityIdentifier`?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift]: Both works for automation, I think that `label` is also visible for VoiceOver, so beneficial to disabled people. Should be localized then, but something is better than nothing.

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift]: Feel like a bit messy adding unlocalized strings to the app. Why be consistent use  accessibilityIdentifier.

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift]: True, but on most recent controls I used label, right now its 160 `identifier` to 50 `label`, so identifier is indeed more frequent to be used.

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Collections/ConversationImagesViewController.swift]: 😔 Ok let keep this for now and create a ticket for cleaning this up.

**Pull request merged**